test/views/MaxPasscodeAttemptsExceededPageSpec.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import org.jsoup.Jsoup
+import views.html.pages.error.maxPasscodeAttemptsExceeded
+
+class MaxPasscodeAttemptsExceededPageSpec extends VatRegViewSpec {
+
+  val view = app.injector.instanceOf[maxPasscodeAttemptsExceeded]
+  implicit val doc = Jsoup.parse(view().body)
+
+  object ExpectedContent {
+    val title = "You need to start again - Register for VAT - GOV.UK"
+    val heading = "You need to start again"
+    val p1 = "This is because you have entered the wrong code too many times."
+    val p2 = "To start again you need to sign out."
+  }
+
+  "MaxPasscodeAttemptsExceededPage" should {
+    "have the correct page title" in new ViewSetup {
+      doc.title mustBe ExpectedContent.title
+    }
+    "have the right header" in new ViewSetup {
+      doc.heading mustBe Some(ExpectedContent.heading)
+    }
+    "have the correct text in the primary paragraph" in new ViewSetup {
+      doc.select(Selectors.p(1)).text must include(ExpectedContent.p1)
+    }
+    "have the correct text in the secondary paragraph" in new ViewSetup {
+      doc.select(Selectors.p(1)).text must include(ExpectedContent.p2)
```

Comment:
You've added an ID to this paragraph, you can use that to select just that specific paragraph rather than all paragraphs on the page. You could even add an ID to the other paragraph of text and do the same.